src/main/java/gov/nasa/pds/api/engineering/controllers/MyProductsApiBareController.java

```diff
@@ -93,6 +94,36 @@ public MyProductsApiBareController(ObjectMapper objectMapper, HttpServletRequest
 	        
     }
     
+    protected Products getLIDVIDs(List<String> lidvids, List<String> fields, HashSet<String> uniqueFields, boolean onlySummary) throws IOException
+    {
+    	String[] aFields = new String[fields == null ? 0 : fields.size()];
+    	if (fields != null)
+    	{
+    		for (int i = 0 ;  i < fields.size(); i++) aFields[i] = ElasticSearchUtil.jsonPropertyToElasticProperty(fields.get(i));
+    	}
+
+    	BoolQueryBuilder find_lidvids = QueryBuilders.boolQuery();
```

Comment:
The code related to the construction of elasticsearch request can be moved to https://github.com/NASA-PDS/registry-api-service/blob/master/src/main/java/gov/nasa/pds/api/engineering/elasticsearch/ElasticSearchRegistrySearchRequestBuilder.java